src/Components/Publishing/Sections/Text.tsx

```diff
@@ -72,53 +79,89 @@ export class Text extends Component<Props, State> {
     return false
   }
 
-  transformNode = (node: Element, index) => {
-    const { color } = this.props
+  renderLinkWithToolTip = (node: Element, index) => {
     // Dont include relay components unless necessary
     // To avoid 'regeneratorRuntime' error
     const LinkWithTooltip = require("../ToolTip/LinkWithTooltip").default
+    const linkNode = node.childNodes[0]
+
+    if (linkNode && linkNode.textContent) {
+      const href = node.getAttribute("href")
+      const { color } = this.props
+      const props = { key: href + index, url: href, color }
+      const next = node.nextSibling
+      const text = linkNode.textContent
+
+      // Check to see if there's an apostrophe following a linked section of
+      // text and if found, return it.
+      const apostrophe = get(next, n => {
+        const str = n.textContent.substr(0, 2)
+        if (/[’'][a-zA-Z]/.test(str)) {
+          return str
+        }
+      })
+
+      if (apostrophe) {
+        // Remove the apostrophe from the original text
+        next.textContent = next.textContent.substring(2)
+        // And wrap the whole thing with a span preventing whitespace breaks
+        return (
+          <span
+            className="preventLineBreak"
+            key={`apostropheLinkNode-${props.key}`}
+          >
+            <LinkWithTooltip {...props}>{text}</LinkWithTooltip>
+            {apostrophe}
+          </span>
+        )
+      }
 
+      return <LinkWithTooltip {...props}>{text}</LinkWithTooltip>
+    }
+  }
+
+  isArtistFollow = (node: Element) => {
+    return (node.getAttribute("class") || "").includes("artist-follow")
+  }
+
+  artistIdToArtistFollow = (artistId: string) => {
+    const data = this.context.tooltipsData
+    if (!data || !data.artists) return null
+    return data.artists[artistId]
+  }
+
+  renderArtistFollowButton = (node: Element) => {
+    // Dont include relay components unless necessary
+    // To avoid 'regeneratorRuntime' error
```

Comment:
Does this still happen?

Reply:
Yes, in positron where we don't have relay